### PR TITLE
First Iteration

### PR DESCRIPTION
**Pull request by 0916dhkim**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,10 +8,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js
-        uses: actions/setup-node@v1
```
Comment (0916dhkim) [inline .github/workflows/ci.yml:11]: - Formatted by prettier.
- Bump `setup-node` action to v3.

File: README.md [hunk 2]
```diff
@@ -48,3 +48,9 @@ yarn prettier --write .
 ```
 yarn start
 ```
+
+### Testing
+
+```
+yarn test
+```
```
Comment (0916dhkim) [inline README.md:56]: Add testing instructions

File: package.json [hunk 3]
```diff
@@ -22,12 +22,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
```
Comment (0916dhkim) [inline package.json:30]: Disable eslint completely.

File: src/HatchwaysBlog.test.js [hunk 4]
```diff
@@ -76,6 +76,28 @@ test("Changing page size from 15 to 25 should render 25 posts", async () => {
   }
 });
 
+test("Page 1 is initially selected", async () => {
```
Comment (0916dhkim) [inline src/HatchwaysBlog.test.js:79]: Test `aria-current` here, so that candidates do not get surprised by private tests.

**Review by bonnieli — APPROVED** [review]
great improvements!

**Pull request merged**